example/README.md

```diff
@@ -114,16 +124,17 @@ mkShell {
 }
 ```
 
-As soon as you save, `lorri` starts building the changes.
+Then tell the daemon to build the changes:
 
+```bash
+$ lorri _ping $(readlink -f shell.nix)
+```
 
-## Change the environment of your shell with lorri & direnv
 
-Keep `lorri watch` running and open up a second terminal. `cd` back
-into `lorri/example`.
+## Change the environment of your shell with lorri & direnv
 
-You should see the direnv warning message again (`error .envrc is
-blocked …`). Type
+Still keeping `lorri daemon` running, let's enable the `direnv` integration. In
```

Comment:
```suggestion
Let's enable the `direnv` integration. In the same terminal, run:
```

Let’s assume the user is following the tutorial step-by-step (plus, lorri should display warnings if the daemon is not running anymore).